README.md

```diff
@@ -8,3 +8,48 @@ This project is not setup for contributing code. It is intended only for trackin
+|  | [AKS Engine](https://github.com/Azure/aks-engine) <br> ([releases](https://github.com/Azure/aks-engine/releases)) | Self-managed clusters on Azure | Azure: incubating <br> API: N/A | [#aks-engine-users](https://kubernetes.slack.com/archives/CU3N85WJK) <br> [GitHub issues](https://github.com/Azure/aks-engine/issues) | [Azure Kubernetes Service](https://docs.microsoft.com/en-us/azure/aks/) <br> [Azure Stack Hub](https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-kubernetes-aks-engine-overview) |
+|  | [Cluster API Azure Provider](https://sigs.k8s.io/cluster-api-provider-azure) <br> ([releases](https://github.com/kubernetes-sigs/cluster-api-provider-azure/releases)) <br> [Tests](https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api-provider-azure) | Self-managed clusters on Azure using Cluster API | CNCF: incubating <br> API: v1alpha2 | [#cluster-api-azure](https://kubernetes.slack.com/archives/CEX9HENG7) <br> [kubernetes-sig-cluster-lifecycle@googlegroups.com](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle) <br> [GitHub issues](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues) |  |
+| Kubernetes Azure Cloud Provider |  |  |  |  |
+|  | [In-tree (legacy)](https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/legacy-cloud-providers/azure) <br> ([releases](https://github.com/kubernetes/kubernetes/releases)) <br> [Tests](https://testgrid.k8s.io/provider-azure-upstream) | Use Azure infra for K8s | CNCF: graduated <br> API: stable | [#provider-azure](https://kubernetes.slack.com/archives/C5HJXTT9Q) <br> [kubernetes-provider-azure@googlegroups.com](https://groups.google.com/forum/#!forum/kubernetes-provider-azure) <br> [GitHub issues](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%2Fprovider%2Fazure+) | [Azure Kubernetes Service](https://docs.microsoft.com/en-us/azure/aks/) |
+|   | [Out-of-tree Provider](https://sigs.k8s.io/cloud-provider-azure) <br> ([releases](https://github.com/kubernetes-sigs/cloud-provider-azure/releases)) <br> [Tests](https://testgrid.k8s.io/provider-azure-cloud-provider-azure) | Use Azure infra for K8s | CNCF: N/A <br> API: alpha | same slack and email <br> [GitHub issues](https://github.com/kubernetes-sigs/cloud-provider-azure/issues) | [Use with AKS Engine](https://github.com/kubernetes-sigs/cloud-provider-azure/blob/master/docs/cloud-controller-manager.md) |
+|   | [Azure Disk CSI Driver](https://sigs.k8s.io/azuredisk-csi-driver) <br> ([releases](https://github.com/kubernetes-sigs/azuredisk-csi-driver/releases)) <br> [Tests](https://testgrid.k8s.io/provider-azure-azuredisk-csi-driver)  | Enable use of Azure disk volume | CNCF: N/A <br> API: v0.6.0 | same slack and email <br> [GitHub issues](https://github.com/kubernetes-sigs/azuredisk-csi-driver/issues) | [Use with AKS Engine](https://github.com/kubernetes-sigs/cloud-provider-azure/blob/master/docs/cloud-controller-manager.md) |
```

Comment:
Here the API version for these CSI drivers are the same as the release. Is that intentional? What does API versions mean here? Perhaps we should leave that out. cc @andyzhangx to confirm

Reply:
should be `Version` other than `API`

Reply:
ack. so `API: N/A`

Reply:
Ack

Reply:
Is there some other pertinent attribute of the project we could put here that would help a user decide on whether it makes sense to use in production?